test/test_embeddings.py

```diff
@@ -1,12 +1,17 @@
 
 import unittest
-
 import numpy as np
-
+import numpy.testing as test
 from word2gauss.embeddings import GaussianEmbedding, text_to_pairs
+import vocab as v
```

Comment:
It's easier to follow the code if you explicitly import the class you'll use: `from vocab import Vocabulary`.  Otherwise someone else needs to remember that `v` refers to `vocab`.